conf/application.conf

```diff
@@ -126,6 +126,7 @@ urls {
   signOut       = "http://localhost:9514/feedback/PODS"
   administratorOrPractitioner = "http://localhost:8204/manage-pension-schemes/administrator-or-practitioner"
   youNeedToRegisterPage = "http://localhost:8204/manage-pension-schemes/you-need-to-register"
+  yourPensionSchemes = "http://localhost:8204/manage-pension-schemes/your-pension-schemes"
```

Comment:
Check app config base has been updated with this new config item and raise PR if not